arm-cognitiveservices/2017-04-18/swagger/cognitiveservices.json

```diff
@@ -0,0 +1,1055 @@
+          "$ref": "#/definitions/CognitiveServicesAccountPropertiesCreateParameters",
+          "description": "Must exist in the request. Must not be null."
+        }
+      },
+      "required": [
+        "sku",
+        "kind",
+        "location",
+        "properties"
+      ],
+      "description": "The parameters to provide for the account."
+    },
+    "CognitiveServicesAccountKind": {
+      "type": "string",
+      "description": "Required. Indicates the type of cognitive service account.",
+      "enum": [
+        "Academic",
+        "Bing.Autosuggest",
+        "Bing.Search",
+        "Bing.Speech",
+        "Bing.SpellCheck",
+        "ComputerVision",
+        "ContentModerator",
+        "CustomSpeech",
+        "Emotion",
+        "Face",
+        "LUIS",
+        "Recommendations",
+        "SpeakerRecognition",
+        "Speech",
+        "SpeechTranslation",
+        "TextAnalytics",
+        "TextTranslation",
+        "WebLM"
+      ],
+      "x-ms-enum": {
+        "name": "Kind",
+        "modelAsString": true
+      }
+    },
+    "CognitiveServicesAccountUpdateParameters": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Required. Gets or sets the SKU of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+        }
+      },
+      "description": "The parameters to provide for the account."
+    },
+    "Sku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the sku name. Required for account creation, optional for update.",
+          "enum": [
+            "F0",
+            "P0",
+            "P1",
+            "P2",
+            "S0",
+            "S1",
+            "S2",
+            "S3",
+            "S4",
+            "S5",
+            "S6"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the sku tier. This is based on the SKU name.",
+          "enum": [
+            "Free",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "The SKU of the cognitive services account."
+    },
+    "CognitiveServicesAccountPropertiesCreateParameters": {
```

Comment:
This is just a placeholder actually. The RP SDK requires this field but we don't have any properties in it.